rules/dropbox_rules/dropbox_external_share.py

```diff
@@ -0,0 +1,25 @@
+from panther_base_helpers import deep_get
+
+ALLOWED_DOMAINS = ["example.com"]
```

Comment:
I would recommend setting this to the empty array so that it can ship without config. Incoming comments for how to work with the empty array into the testing suite to show that it works as intended.